lib/client/api.go

```diff
@@ -2066,6 +2076,23 @@ func PlayFile(ctx context.Context, tarFile io.Reader, sid string) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	// Return errors if this is desktop, app, db or unknown session.
+	if len(sessionEvents) > 0 {
+		switch typ := sessionEvents[0].GetType(); typ {
```

Comment:
Now that we have the same code in 2 places, I suggest we extract this to a `func checkPlayable(firstEvent apievents.AuditEvent) error` function and use it in both places.

Reply:
We'd have to do logic to determine if we can show the desktop url or not.  Since the `tsh play file` doesn't require a Teleport client it shows a different desktop error.  Do we still want to consolidate?

Reply:
I see. This is fine then.